statistics/handle/handle.go

```diff
@@ -688,7 +688,7 @@ func (h *Handle) FlushStats() {
 		}
 	}
 	if err := h.DumpStatsDeltaToKV(DumpAll); err != nil {
-		logutil.BgLogger().Error("[stats] dump stats delta fail", zap.Error(err))
+		logutil.BgLogger().Error("[stats] dump stats all fail", zap.Error(err))
```

Comment:
Why?

Reply:
> Why?

There call ``h.DumpStatsDeltaToKV(DumpAll)`` but log message is delta